packages/react/src/components/TooltipIcon/TooltipIcon.js

```diff
@@ -148,7 +150,7 @@ TooltipIcon.propTypes = {
    * Specify an icon as children that will be used as the tooltip trigger. This
    * can be an icon from our Icon component, or a custom SVG element.
    */
-  children: PropTypes.node.isRequired,
+  children: PropTypes.node,
```

Comment:
Do we need to add a deprecation notice? Or do we plan to still allow the old functionality, but just surface the new `renderIcon` prop in addition?

Reply:
I mentioned that it can be deprecated but I didn't make any changes to deprecate it. I'm not sure how things will change with the Popover still under development